### PR TITLE
Extend the README

### PR DESCRIPTION
**Pull request by mikhailshilkov**

### CODE REVIEW

File: sdk/python/README.md [hunk 1]
```diff
@@ -1,3 +1,91 @@
 # Native Google Cloud Pulumi Provider (preview)
 
-The native Google Cloud Provider for Pulumi lets you use Google Cloud resources in your cloud programs.
+The native Google Cloud Provider for Pulumi lets you provision Google Cloud resources in your cloud programs.
```
Comment (viveklak) [inline sdk/python/README.md:3]: Is there a requirement to have an sdk/python/README.md? Wasn't sure why we create it generally.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,91 @@
...
+Credentials configuration is compatible with the classic GCP provider.
+
+Please refer to [this quickstart guide](
+https://www.pulumi.com/docs/intro/cloud-providers/gcp/setup/) for possible configuration options.
+
+## Building
+
+### Dependencies
+
+- Go 1.15
```
Comment (viveklak) [inline README.md:64]:
```suggestion
- Go 1.16
```
Looks like we are already on go 1.16

**Review by viveklak — APPROVED** [review]

File: sdk/python/README.md (hunk 1 above)
Comment (stack72) [reply sdk/python/README.md:3]: The Python README is the one that shows in pypi - so we *generally* copy it 🤷

File: sdk/python/README.md [hunk 3]
```diff
@@ -1,3 +1,91 @@
...
+Credentials configuration is compatible with the classic GCP provider.
+
+Please refer to [this quickstart guide](
+https://www.pulumi.com/docs/intro/cloud-providers/gcp/setup/) for possible configuration options.
+
+## Building
+
+### Dependencies
+
+- Go 1.15
```
Comment (viveklak) [inline sdk/python/README.md:64]:
```suggestion
- Go 1.16
```

Comment (mikhailshilkov) [reply sdk/python/README.md:64]: It's currently Go 1.15 in the SDK

**Pull request merged**